<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="jenkins-agent-10-25-197-175" name="com.mypurecloud.sdk.v2.ApiClientRetryTest" tests="12" failures="0" timestamp="04 Dec 2021 07:32:59 GMT" time="70.094" errors="0">
  <testcase name="invokeTestWith_502" time="13.078" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_200" time="0.002" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_502" time="1.002" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_504" time="2.067" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_429" time="6.236" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_429_And_No_MaxRetryTime" time="0.104" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_429" time="1.003" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_503_And_RetryConfig" time="3.008" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_504_And_No_RetryAfter_Header" time="3.001" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_504_No_MaxRetryTime" time="0.122" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="shouldRetryTestWith_502_And_0_MaxRetryTime" time="0.356" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
  <testcase name="invokeTestWith_503" time="40.115" classname="com.mypurecloud.sdk.v2.ApiClientRetryTest"/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryTest -->
